pages/linux/edquota.md

```diff
@@ -0,0 +1,23 @@
+# edquota
+
+> Edit quota for a user or group.
+
+- Edit the user quota (default):
+
+`edquota --user {{user}}`
+
+- Edit the group quota:
+
+`edquota --group {{group}}`
+
+- Perform operations on given file system ONLY (default is to perform operations on all file systems with quota):
```

Comment:
Sounds good -- only a minor nitpick: I'd split the two lines after the second period, rather than the first :) Like this:

> Edit quota for a user or group. By default it operates on all file systems with quotas.
> Quota information is stored permanently in the quota.user and quota.group files in the root of the file system.